src/mkdocstrings/handlers/base.py

```diff
@@ -390,6 +390,8 @@ class Handlers:
     this for the purpose of caching. Use [mkdocstrings.plugin.MkdocstringsPlugin.get_handler][] for convenient access.
     """
 
+    current_page: Optional[str] = None
```

Comment:
Honestly at this point I'm really abusing the fact that `Handlers` happens to be available to both the plugin and the extension. Maybe later I'll see a better way but right now I don't see.

Reply:
Don't worry, putting data in-between the plugin and the extension was the primary design :slightly_smiling_face: It's impossible without. But yeah, maybe later, with more hindsight we will see a way to refactor elegantly.